lib/thumbs/pull_request_worker.rb

```diff
@@ -920,6 +920,10 @@ def comment_code_approvals
       code_reviews
     end
 
+    def remove_build_dir
+      FileUtils.mv(@build_dir, "#{@build_dir}.#{DateTime.now.strftime("%s")}")
```

Comment:
Nope, this will move the directory to a new name, 
/tmp/thumbs/build_guid gets moved to directory /tmp/thumbs/build_guid.timestamp
So completely separate directories.
But for cleanliness and future dealing with the output cruft, I agree it should be in a special place like /tmp/thumbs/archive.